linux/keyman-config/km-config

```diff
@@ -13,6 +15,7 @@ if __name__ == '__main__':
     parser.add_argument('--version', action='version', version='%(prog)s version '+__version__)
     parser.add_argument('-v', '--verbose', action='store_true', help='verbose logging')
     parser.add_argument('-vv', '--veryverbose', action='store_true', help='very verbose logging')
+    parser.add_argument('-i', '--install', action='store', nargs='?', help='install .kmp package')
```

Comment:
Is there a man page to add this? I looked at linux/keyman-config/README.md and saw there's a command-line
> km-package-install -f <kmp file> install Keyman keyboard package from a local .kmp file

Reply:
Good point. I forgot to check for a man page.

Turns out that the possible arguments get added dynamically during the package build - there's only a template man file with the description (`linux/keyman-config/maninc/km-config.inc`).

`km-package-install` will install the keyboard without displaying the keyboard info dialog first.